AsyncDisplayKit/Details/Transactions/_ASAsyncTransactionGroup.h

```diff
@@ -8,19 +8,20 @@
 //  of patent rights can be found in the PATENTS file in the same directory.
 //
 
-#import <UIKit/UIKit.h>
```

Comment:
I see, removing this is a great change.  If the UIApplication dependency is needed, we should probably add a comment next to the import in a future diff to point out its utility.